scripts/zones/Lufaise_Meadows/Zone.lua

```diff
@@ -17,10 +17,16 @@ zoneObject.onInitialize = function(zone)
 
     SetServerVariable("realPadfoot", math.random(1, 5))
     for _, v in pairs(ID.mob.PADFOOT) do
-        SpawnMob(v)
+        local respawnP = GetServerVariable("\\[SPAWN\\]"..ID.mob.PADFOOT)
```

Comment:
Sanity checks are failing on this line. I believe you might need to replace the `ID.mob.PADFOOT` with just `v`

Reply:
+1. This is saving the table "to string" representation to DB instead of an ID, so GetSpawn() will never get anything and padfoot wont't respawn